component/src/test/java/io/siddhi/extension/execution/math/AtanFunctionExtensionTestCase.java

```diff
@@ -146,23 +141,22 @@ public void exceptionTestCase3() throws Exception {
             public void receive(long timeStamp, Event[] inEvents,
                                 Event[] removeEvents) {
                 EventPrinter.print(timeStamp, inEvents, removeEvents);
+                for (Event event : inEvents) {
+                    AssertJUnit.assertEquals(null, event.getData(0));
+                }
             }
         });
         InputHandler inputHandler = siddhiAppRuntime
                 .getInputHandler("InValueStream");
         siddhiAppRuntime.start();
         inputHandler.send(new Object[]{null});
         Thread.sleep(100);
-        AssertJUnit.assertTrue(appender.getMessages().contains("Input to the math:atan() function cannot be null"));
```

Comment:
Assert for events arrived, Just to make sure what events came to the callback.
Need to do that for all modified test cases